benefit/data/sheets/ComplexTypes/ComplexTypes.json

```diff
@@ -437,6 +437,8 @@
   {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReq", "ListElementCode": "hearingRecordingRequest", "FieldType": "hearingRecordingRequest", "ElementLabel": "Hearing recording request", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReq", "ListElementCode": "otherPartyId", "FieldType": "Text", "ElementLabel": "Hearing ID", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReqUi", "ListElementCode": "otherPartyName", "FieldType": "Text", "ElementLabel": "Other party name", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReqUi", "ListElementCode": "hearingRecordingStatus", "FieldType": "DynamicList", "ElementLabel": "Status", "SecurityClassification": "PUBLIC"}
-
+  {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReqUi", "ListElementCode": "hearingRecordingStatus", "FieldType": "DynamicList", "ElementLabel": "Status", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "otherPartyOptionName", "FieldType": "Text", "ElementLabel": "Other party name", "SecurityClassification": "PUBLIC"},
```

Comment:
Maybe it doesn't matter but normally the ID's start with lower case letter

Reply:
it doesn't, as I tested and it worked. I followed the SscsDocument complex type example